src/05-derivatives/01-introduction.md

```diff
@@ -28,6 +28,159 @@ in [Derived dataset and pipeline description][derived-dataset-description].
     to ensure that the values are still relevant and appropriate to the type of
     output data.
 
+## Model-based grouping of derivatives
+
+Here, a *model* is defined as a process via which parameters that describe
+the data are specified (consistent with
+[this definition](https://plato.stanford.edu/entries/models-science/#DescEqua)).
+This could include, but is not limited to things like:
+
+1.  The beta weights for different kinds of events in an event-related design
+    in fMRI.
+
+1.  The six components of the diffusion tensor in DTI.
+
+1.  Etc.
+
+Model-based derivatives SHOULD be saved in a directory named `model-<model_id>`
+that is placed under the datatype from which the model was derived.
+
+{{ MACROS___make_filetree_example(
+
+   {
+   "<pipeline_name>": {
+      "sub-<label>": {
+         "<datatype>": {
+            "model-<label>": {
+                "<model files>": ""
+            }
+         }
+      }
+   }
+   }
+
+) }}
+
+The specification of `<model files>` is [introduced below](#file-naming-conventions).
+
+### Models file
+
+Template:
+
+{{ MACROS___make_filetree_example(
+
+   {
+   "<pipeline_name>": {
+      "sub-<label>": {
+         "<datatype>": {
+            "model-<label>": {
+                "<model files>": ""
+            },
+            "sub-<label>_models.tsv": "",
+            "sub-<label>_models.json": ""
```

Comment:
Here, in singular?

```suggestion
            "sub-<label>_model.tsv": "",
            "sub-<label>_model.json": ""
```

Reply:
(I don't think so, see above)

Reply:
OIC. I got confused. The description makes it sound like there is one `models.tsv` file for the dataset. Here, we introduce the notion that different subjects (And different datatypes?) can have their own `sub-<label>_models.tsv` file (and `<datatype>_models.tsv` is also allowable?). We might want to describe this before providing an example.

Reply:
Right, when I added the `sub-<>_` prefix I actually disallowed a `/models.tsv` implicitly. I'll send some changes to make this explicit.